image-patching/stopgap-script/nuage_overcloud_full_patch.py

```diff
@@ -276,10 +281,27 @@ def main(args):
             handler = logging.FileHandler(argsDict['logFile'][0])
             handler.setFormatter(formatter)
             logger.addHandler(handler)
+        if 'RpmPublicKey' not in argsDict and 'no-signing-key' not in argsDict:
+
+            logger.error("'--RpmPublicKey' or '--no-signing-key' are not passed in image patching command, Either of those parameters is mandatory")
+            sys.exit()
+            
+        if 'RpmPublicKey' in argsDict:
+            file_exists = os.path.isfile(argsDict['RpmPublicKey'][0])
+            if file_exists:
+               pass
```

Comment:
copy thr gpgkey instead of pass

Reply:
1) Create global variables GPGCHECK=0 and GPGKEY=''
2) Set them to proper values if RpmPublicKey' is provided.